### PR TITLE
dive deeper for structs

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: public/content/en/basics.md [hunk 1]
```diff
@@ -416,6 +416,10 @@ Note that a `struct` can't inherit from another `struct`.
 Hierachies of types can only be built using classes
 which we will see in a future section.
 
+### Dive deeper
+
+- [Structs in details](https://dlang.org/spec/struct.html)
```
Comment (stonemaster) [inline public/content/en/basics.md]: "in detail"

**Comment by wilzbach** [comment]
Rebased, but I am not so sure how good only one item in a link list looks. Do you have a better idea?

File: public/content/en/basics.md [hunk 2]
```diff
@@ -416,6 +416,10 @@ Note that a `struct` can't inherit from another `struct`.
 Hierachies of types can only be built using classes
 which we will see in a future section.
 
+### Dive deeper
```
Comment (stonemaster) [inline public/content/en/basics.md]: In-depth like in the other PRs

**Comment by wilzbach** [comment]
Rebased

**Pull request merged**